src/components/ListingDetails/index.tsx

```diff
@@ -118,7 +121,8 @@ const docFields: ListingField<DocumentTranslation>[] = [
   // Languages
   {
     label: 'Language(s)',
-    getValue: data => data.languages.join(', '),
+    getValue: data =>
+      data.languages.length > 0 ? data.languages.join(', ') : 'Not Available',
```

Comment:
if language is required for DOC, do we not need this check?

Reply:
ur so right, will remove

Reply:
hmm probably not? but better to have this in case Retool form has any bugs and they somehow make a listing with no language requirements